src/effects/VST3/VST3Effect.cpp

```diff
@@ -339,32 +339,129 @@ bool VST3Effect::SupportsAutomation() const
    return false;
 }
 
-bool VST3Effect::SaveSettings(
-   const EffectSettings &, CommandParameters & parms) const
+struct VST3Effect::ParameterInfo
 {
-   if(mEditController == nullptr)
+   Steinberg::Vst::ParameterInfo mVstParamInfo;
+};
```

Comment:
Maybe this `struct` should just simplify to a `using` alias

Reply:
done in commit `simplified parameterInfo`